README.md

```diff
@@ -152,15 +146,65 @@ Use <kbd>Ctrl + Shift + F</kbd> to see auto-detected subtitle tracks for streami
 
 <kbd>Ctrl + Shift + F</kbd> also allows you to load an empty subtitle track so that you can extract audio and screenshots from streaming video without loading subtitles.
 
-### Extension side panel UI
+#### Filtering subtitle text
+
+If you'd like to filter out specific instances subtitle text, one way to do so is by using a regular expression (regex). asbplayer can match any sequence following a specified regex pattern and remove the matches.
+
+Under the MISC section in asbplayer settings, locate the "Subtitle regex filter" textbox. Enter an appropriate regex to filter desired content.
+You can replace filtered content similarly by entering a string into the "Subtitle regex filter text replacement" textbox. Leaving this blank will simply remove the content.
+
+Potential use cases for filtering include:
+ - Remove names enclosed by parenthesis to indicate speakers  (i.e. "**（山田）**　元気ですか？")
+ - Remove indications enclosed by brackets that sound or music that is playing (i.e. "**\[PLAYFUL MUSIC]**")
+
+Helpful materials on writing regular expressions:
+- [GeeksForGeeks regex tutorial](https://www.geeksforgeeks.org/write-regular-expressions/)
+- [RexEgg regex cheatsheet](https://www.rexegg.com/regex-quickstart.html)
+- Test regex with the [Regexr webtool](https://regexr.com/)
+
+#### Downloading subtitle files
+
+Once loaded into the extension, you can choose to download subtitles by opening the side panel and clicking the `Download Subtitles as SRT` button in the top right. The same can be done with subtitles loaded into the webtool by accessing the [asbplayer website](https://killergerbah.github.io/asbplayer/) and clicking the same download button in the top left.

```

Comment:
Suggestion to shorten and use "website" instead of "webtool": 
"Once loaded into the extension, you can download the subtitles by opening the side panel and clicking the `Download Subtitles as SRT` button in the top-right. You can also download subtitles via the website by clicking the same download button in the top-left."

I would also remove the link to the website. Linking the website here may be a bit misleading since subtitles can't be downloaded simply by just accessing the website.